mlir/include/mlir/IR/VectorTypes.h

```diff
@@ -0,0 +1,47 @@
+//===- VectorTypes.h - MLIR Vector Types ------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef MLIR_DIALECT_VECTOR_IR_VECTORTYPES_H_
+#define MLIR_DIALECT_VECTOR_IR_VECTORTYPES_H_
```

Comment:
Should these be like the following? I see that the preprocessor guards in this directory are a bit inconsistent.
```
#ifndef MLIR_IR_VECTORTYPES_H
#define MLIR_IR_VECTORTYPES_H
```